build-tools/xamarin-android-docimporter-ng/Xamarin.Android.ApiTools.ParameterNameExtractor/Xamarin.Android.ApiTools.JavaStubImporter/JavaStubParser.cs

```diff
@@ -345,7 +347,7 @@ public JavaStubGrammar ()
 				type.Name = (string)node.ChildNodes [3].AstNode;
 				type.Deprecated = ((IEnumerable<string>)node.ChildNodes [0].AstNode).Any (v => v == "java.lang.Deprecated" || v == "Deprecated") ? "deprecated" : "not deprecated";
 				type.TypeParameters = isEnum ? null : (JavaTypeParameters) node.ChildNodes [4].AstNode;
-				type.Members = (IList<JavaMember>) node.ChildNodes [7].AstNode;
+				type.Members = (IList<JavaMember>)node.ChildNodes [isEnum ? 6 : 7].AstNode;
```

Comment:
This is the first I've looked at this project, so forgive me if I'm way off.

There are a lot of "position-based" lookups like this in here. Is there a way to lookup a child node based on its "name" or "type" instead of using magic numbers?